### DISCUSSION TITLE
Overview page

### DISCUSSION THREAD

**Discussion by tronauts** [jomjol/AI-on-the-edge-device#2299] [Q&A]
after step 6 and reboot, why doesn't the overview page appear? and return to the initial setup page

<img width="580" alt="image" src="https://user-images.githubusercontent.com/82866068/230778979-e1b79bd0-1f02-4b59-84b7-3a1bb09c7921.png">

**Comment by caco3** [comment]
Maybe it failed to save the https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-setupmode

Try again or set it to false manually.
